### ISSUE TITLE
Funding and publishing opportunities

### ISSUE THREAD

**Issue by vanatteveldt** [ccs-amsterdam/digitaltrackingworkshop18#11]
Maybe we should also discuss what kind of (collaborative) funding opportunities there are, and whether it makes sense to work to something like a special issue or something related?

**Comment by magdalenawojcieszak** [comment]
YES! I know that the NWO had some calls for NL-US collaboration specifically. Should we explore it? Does anyone know the name of that particular call?

**Comment by vanatteveldt** [comment]
I think that was the Trans-Atlantic Platform "Digging into Data" Challenge, but I don't that is a recurring call. Their website does say they have a call coming up on 'social innovation', but that's probably not as relevant: https://www.transatlanticplatform.com/
